docs/more-pushmsg.md

```diff
@@ -193,6 +193,77 @@ public class GotifyClient {
 }
 ```
 
+### VB/VBA
+
+```vb
+' --- Based on pushover-vba by Mauricio Arieira (https://github.com/makah/pushover-vba)
+
+Const GOTIFY_URL As String = "https://gotify.some-web.space/message?token=<apptoken>"
+
+' Send a post via PushOver to Gotify server
+' @param In title as String: The message title
+' @param In message as String: The message that you want to send
+' @param In priority as Integer: The message priority
+' @return as String: The post response
+Public Function PushToGotify(ByVal title As String, ByVal message As String, ByVal priority As Integer) As String
+    Dim xhttp As Object, params As String
+
+    params = StringFormat("title={0}&message={1}&priority={2}", title, message, priority)
+
+    Set xhttp = CreateObject("MSXML2.ServerXMLHTTP")
+    With xhttp
+        .Open "POST", GOTIFY_URL, False
+        .setRequestHeader "Content-type", "application/x-www-form-urlencoded"
+        .Send params
+        PushToGotify = .responseText
+    End With
+End Function
+
+' ----- copied from ExcelUtils.
+' ----- from: https://github.com/makah/ExcelHelper
+'
+' Generate a string using .NET format, i.e. {0}, {1}, {2} ...
+' @param In strValue as String: A composite format string that includes one or more format items
+' @param In arrParames as Variant: Zero or more objects to format.
+' @return as String: A copy of format in which the format items have been replaced by the string representations of the corresponding arguments.
+' @example: Debug.Print StringFormat("My name is {0} {1}. Hey!", "Mauricio", "Arieira")
+Public Function StringFormat(ByVal strValue As String, ParamArray arrParames() As Variant) As String
```

Comment:
Hey, thanks for your contribution! This code seems pretty bloaty with all the comments and extra string format function. Also, I think it is enough to describe one way to send a message. Could you remove the string format function and the extra information about sending with json?

Reply:
Thank you for the comment. I cleaned up the code by your recommendations. I hope it's okay to leave the link to the code origin in the comment.